devel/fortio/fortioLib/http.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fortio
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+)
+
+// ExtraHeaders to be added to each request.
+var extraHeaders http.Header
+
+// Host is treated specially, remember that one separately.
+var hostOverride string
+
+func init() {
+	extraHeaders = make(http.Header)
+	extraHeaders.Add("User-Agent", userAgent)
+}
+
+// Verbose controls verbose/debug output, higher more verbose.
+var Verbose int
```

Comment:
I know `glog` uses a numerical verbosity level rather than the usual on/off true/false approach.  @ldemailly any reason you used the `log` package over the `glog` package?  In at least pilot we use `glog` everywhere